components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/ApiDAO.java

```diff
@@ -167,8 +167,8 @@
      * @return {@code List<API>} matching results
      * @throws APIMgtDAOException if error occurs while accessing data layer
      */
-    List<API> attributeSearchAPIsStore(List<String> roles, Map<String, String> attributeMap,
-                                       int offset, int limit) throws APIMgtDAOException;
+    List<API> searchAPIsByAttribute(List<String> roles, Map<String, String> attributeMap,
+                                    int offset, int limit) throws APIMgtDAOException;
```

Comment:
Fixed in : https://github.com/wso2/carbon-apimgt/pull/4049/commits/406333a806f4eaa45888c9a250d017136d224abf